### DISCUSSION TITLE
When using Spring Cloud Gateway 4.1.5 with SkyWalking, the topology is wrong.

### DISCUSSION THREAD

**Discussion by Camio1945** [apache/skywalking#12675] [Q&A]
1. I have put `apm-spring-cloud-gateway-4.x-plugin-9.3.0.jar` and `apm-spring-webflux-6.x-plugin-9.3.0.jar` in `plugins` folder:
![image](https://github.com/user-attachments/assets/3d79ee86-f282-41e2-9698-12b62b513ff3)

2. I have assigned the VMOptions for gateway and other micro services.
![image](https://github.com/user-attachments/assets/e00fb4ae-0dcc-4935-9621-b0ca608bd2b3)

3. The topology is wrong. 
The right route should be `User -> gateway -> other micro services`.
![image](https://github.com/user-attachments/assets/014de20e-e018-40b7-ab07-f713175ab7eb)

What should I do? Does anyone have a working sample using Spring Cloud Gateway 4.1.5?

**Comment by wu-sheng** [comment]
I think this has been answered repeatedly. Spring Gateway 4 is not supported, as no one contributed.
